common/app/implicits/Dates.scala

```diff
@@ -13,13 +16,19 @@ trait Dates {
     def toJoda: JodaDuration = new JodaDuration(duration.toMillis)
   }
 
+  implicit class CapiRichDateTime(cdt: CapiDateTime) {
+    def toJoda: DateTime = new DateTime(cdt.dateTime)
+  }
+
   object Epoch {
     lazy val zero: LocalDate = new LocalDate(0)
     def day(dayOfEpoch: Int): LocalDate = zero.plusDays(dayOfEpoch)
   }
 
   def today(): LocalDate = LocalDate.now()
 
+  def jodaToJavaInstant(date: DateTime): Instant = date.toDate.toInstant
+
```

Comment:
You could use the same implicit mechanism we use above, so you can then call `myDate.toJavaInstant` 
ie:
```
implicit class RichDate(date: DateTime) {
    def toJavaInstant: Instant = date.toDate.instant
}
```

Reply:
Yes agreed - however I wanted to move away from using implicit classes everywhere as its a bit 'black box magicky' and felt this particular use case was better suited as a method.

Reply:
Sounds good to me